dart/test/test_utils.dart

```diff
@@ -378,16 +381,14 @@ void runTest({Codec<List<int>, List<int>> gzip, bool isWeb = false}) {
 
     try {
       throw ArgumentError('Test error');
-    } catch (error, stackTrace) {
+    } catch (error) {
       final eventWithoutContext = SentryEvent(
         eventId: SentryId.empty(),
-        exception: error,
-        stackTrace: stackTrace,
+        throwable: error,
```

Comment:
why are we not testing if caught stack trace?

Reply:
It not really the subject of this test, and as we catch an `Error`, the stacktrace should be in the Error itself.

Reply:
makes sense, I'd just guarantee that another test is testing the `stackTrace` or we'd need to create a new one for this use case

Reply:
✅